src/main/java/org/killbill/billing/client/KillBillClient.java

```diff
@@ -827,15 +827,18 @@ public Payments getPaymentsForAccount(final UUID accountId, final AuditLevel aud
         return httpClient.doGet(uri, queryParams, Payments.class);
     }
 
-    public Payment createPayment(final ComboPaymentTransaction comboPaymentTransaction, final Map<String, String> pluginProperties, final String createdBy, final String reason, final String comment) throws KillBillClientException {
+    public Payment createPayment(final ComboPaymentTransaction comboPaymentTransaction, final Map<String, String> pluginProperties, final String createdBy, final String reason, final String comment, final Multimap<String, String> queryParams) throws KillBillClientException {
```

Comment:
Couple of notes:

* I'd rather have the signature accept a `List` of control plugin names, so the client doesn't have to look-up the query parameter name
* For consistency, `pluginProperties`, `createdBy`, `reason` and `comment` should be the last arguments